code/modules/server_tools/server_tools.dm

```diff
@@ -2,17 +2,28 @@ GLOBAL_VAR_INIT(reboot_mode, REBOOT_MODE_NORMAL)	//if the world should request t
 GLOBAL_PROTECT(reboot_mode)
 
 /world/proc/RunningService()
-	return params[SERVICE_WORLD_PARAM]
+	return CONFIG_GET(string/service_command) != null
 
 /proc/ServiceVersion()
 	if(world.RunningService())
 		return world.params[SERVICE_VERSION_PARAM]
 
 /world/proc/ExportService(command)
-	return RunningService() && shell("python code/modules/server_tools/nudge.py \"[command]\"") == 0
+	if (world.RunningService())
+		var/script = CONFIG_GET(string/service_command)
+		var/status = shell("[script] [command]")
+		log_world("SERVICE COMMAND: [script] [command]")
+		if (status == null)

```

Comment:
You are putting spaces after `if`s, different to the rest of the codebase yo.

Also, can just `!status`?

Reply:
Need to explicitly check it is null and not 0